packages/flutter_tools/lib/src/ios/xcodeproj.dart

```diff
@@ -30,11 +34,11 @@ String _generatedXcodePropertiesPath(String projectPath) {
 
 /// Writes default Xcode properties files in the Flutter project at [projectPath],
 /// if project is an iOS project and such files do not already exist.
-void generateXcodeProperties(String projectPath) {
+Future<void> generateXcodeProperties(String projectPath) async {
   if (fs.isDirectorySync(fs.path.join(projectPath, 'ios'))) {
     if (fs.file(_generatedXcodePropertiesPath(projectPath)).existsSync())
-      return;
-    updateGeneratedXcodeProperties(
+      return new Future<void>.value();
```

Comment:
A plain `return;` should do nicely since you are using `async`.

Reply:
done